app/controllers/api/v1/moves_controller.rb

```diff
@@ -73,6 +74,25 @@ def find_move
           .includes(:from_location, :to_location, person: { profiles: %i[gender ethnicity] })
           .find(params[:id])
       end
+
+      def import_moves_from_nomis
```

Comment:
This can be private right?
My suggestion (to take with a truckload of salt as I'm the new kid on the block here and I'm most probably missing lots of context) would be to extract this behavior in a service object with a well defined interface and defer to this `nomis` service to interact with it.

Reply:
In case: this is an article I found helpful in the past https://www.toptal.com/ruby-on-rails/rails-service-objects-tutorial

Reply:
> This can be private right?
> My suggestion (to take with a truckload of salt as I'm the new kid on the block here and I'm most probably missing lots of context) would be to extract this behavior in a service object with a well defined interface and defer to this `nomis` service to interact with it.

It is already private